conf/workloads/sql/tpcds.conf

```diff
@@ -0,0 +1,28 @@
+#tpcds
+hibench.tpcds.tiny.table                    1
+hibench.tpcds.small.table                   10
+hibench.tpcds.large.table                   1000
+hibench.tpcds.huge.table                    10000
+hibench.tpcds.gigantic.table                50000
+hibench.tpcds.bigdata.table                 100000
```

Comment:
What is the proper running time for each scale? If I have a cluster of 7 slaves, totally 500 cores and 1.5T memory?